bigtable-hbase-integration-tests/src/test/java/com/google/cloud/bigtable/hbase/TestBatch.java

```diff
@@ -366,4 +375,12 @@ public void testBatchGets() throws Exception {
 
     table.close();
   }
+
+  public void testBatchDoesntHang() throws Exception {
+    Connection closedConnection = ConnectionFactory.createConnection(IntegrationTests.getConfiguration());
+    Table table = closedConnection.getTable(TABLE_NAME);
+    closedConnection.close();
+    thrown.expect(RetriesExhaustedWithDetailsException.class);
```

Comment:
The reason not to check for a specific type of exception HERE is that it's not part of this specific test.  If we want to observe specific types of behavior for .close() exceptions, I would ask that we do it somewhere else.  This test is specifically made to test .put() batch operation not specific .close() behavior.